packages/amplication-server/src/core/environment/environment.service.ts

```diff
@@ -0,0 +1,27 @@
+import { Injectable } from '@nestjs/common';
+import { PrismaService } from 'nestjs-prisma';
+
+import {
+  Environment,
+  CreateEnvironmentArgs,
+  UpdateOneEnvironmentArgs
+} from './dto';
+import { FindOneArgs } from 'src/dto';
+
+@Injectable()
+export class EnvironmentService {
+  constructor(private readonly prisma: PrismaService) {}
+
+  async createEnvironment(args: CreateEnvironmentArgs): Promise<Environment> {
+    return this.prisma.environment.create(args);
+  }
+
+  async getEnvironment(args: FindOneArgs): Promise<Environment | null> {
```

Comment:
Why is this method not exposed by the resolver?

Reply:
The current flow requires minimal interaction with the environment. We only need to know its ID in order to send it when calling "CreateDeployment".
I already added in the next PR a FieldResolver on app to get the app environments in order to get the single env ID.

Later down the road, we will implement the rest of the functionality which is required for multi env.